tools/c7n_azure/c7n_azure/resources/advisor.py

```diff
@@ -0,0 +1,44 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('advisor')
+class Advisor(ArmResourceManager):
+    """Azure Advisor Resource
+
+    :example:
+
+    Returns all azure advisor recommendations of type Security
+
+    .. code-block:: yaml
+
+        policies:
+        - name: list-advisor-security-recommendations
+          resource: azure.advisor
```

Comment:
Should it be renamed to `advisor-recommendations`?

Reply:
as recommended, I renamed to advisor-recommendations